### PR TITLE
feat(Resource-status): Add metaservice Resource

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This adds the support for meta-services within Resource status

Capture d’écran 2021-04-02 à 10 18 56
Capture d’écran 2021-04-02 à 10 19 11
Capture d’écran 2021-04-02 à 10 19 39

**Fixes** # (issue)

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 21.04.x (master)

### CODE REVIEW

File: www/front_src/src/Resources/Details/tabs/Metrics/Metrics.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+              <div className={classes.service}>
+                <StatusChip
+                  label={resource.short_type}
+                  severityCode={SeverityCode.None}
+                />
+                <SelectableResourceName
+                  name={resource.name}
+                  onSelect={() => selectResource(resource)}
+                />
+              </div>
```
Comment (kduret) [inline www/front_src/src/Resources/Details/tabs/Metrics/Metrics.tsx:69]: Host name is not displayed ?
Cause service name alone is not enough to identify the resource

**Comment by bdauria** [comment]
@kduret fixed 
Capture d’écran 2021-04-02 à 11 11 24

**Comment by adr-mo** [comment]
Just a small remark it could be nicer IMHO if host/service were moved to the left

**Comment by Thebarda** [comment]
In the Metric tab, the card's content is not centered

**Comment by bdauria** [comment]
@adr-mo done: 
Capture d’écran 2021-04-02 à 17 33 45

**Review by Thebarda — APPROVED** [review]
Tested & LGTM
https://www.loom.com/share/c2cfbd63e33b4ce487a8bcd947ec79c5

**Review by karimsx — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**